lib/gc/Transforms/OneDNNGraphToLinalg.cpp

```diff
@@ -5,6 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+#ifdef GC_HAS_ONEDNN_DIALECT
```

Comment:
Can we not include the file in the build instead?

Reply:
We can conditionally add it to the global property that forms an appropriate library (be that gcpasses or a separate one, no matter). I had a similar problem in #205